README.md

```diff
@@ -76,15 +76,25 @@ API
 
 The public Accumulo API is composed of :
 
- * All public classes and interfaces in the org.apache.accumulo.core.client
-   package, as as well as all of its subpackages excluding those named *impl*.
- * Key, Mutation, Value, Range, Condition, and ConditionalMutation in
-   org.apache.accumulo.core.data.
- * All public classes and interfaces in the org.apache.accumulo.minicluster
-   package, as well as all of its subpackages excluding those named *impl*.
- * Anything with public or protected acccess within any Class or Interface that
-   is in the public API. This includes, but is not limited to: methods, members
-   classes, interfaces, and enums.
+All public types in the following packages and their subpackages excluding
+those named *impl*, *thrift*, or *crypto*. 
+
+   * org.apache.accumulo.core.client
+   * org.apache.accumulo.core.data
+   * org.apache.accumulo.core.security
+   * org.apache.accumulo.minicluster
+
+A type is a class, interface, or enum.  Anything with public or protected
```

Comment:
Haha, yeah, I remember having that same reaction of "really?", but who knows. It wouldn't be the first weird thing I dreamed up. Since you guys took the time to clarify this, let's try to make this as bullet-proof as possible. Sounds like package-protected should be mentioned with private as being inaccessible.